somatic_llr_filter.py

```diff
@@ -0,0 +1,303 @@
+import argparse
+import sys
+import os
+import re
+import vcfpy
+import tempfile
+import csv
+import math
+from collections import OrderedDict
+
+def define_parser():
+# inputs:
+# required:
+# - somatic vcf, with tumor and normal readcounts
+
+# optional:
+#  - sequence error rate (range 0 to 1) - default 0.005
+#  - tumor purity (range 0 to 1) - default 1
+#  - normal contamination rate (range 0 to 1) - default 1
+#  - llr cutoff at which to apply filter - default 1
+#  - field corresponding to allele depth - default AD
+#  - field corresponding to site depth - default DP
+#  - tumor sample name - default TUMOR
+#  - normal sample name - default NORMAL
+
+    parser = argparse.ArgumentParser('vcf-binomial-filter')
+    parser.add_argument(
+        "input_vcf",
+        help="A VCF file with at least two samples (tumor and normal) and readcount information"
+    )
+    parser.add_argument(
+        "output_vcf",
+        help="Path to write the output VCF file"
+    )
+    parser.add_argument(
+        "--sequence-error-rate",
+        help="expected sequencing error rate (range 0 to 1) - default 0.01",
+        type=float,
+        default=0.005
```

Comment:
The default here doesn't match the help text.

Reply:
new commits fix this